src/main/groovy/io/openliberty/tools/gradle/tasks/DeployTask.groovy

```diff
@@ -553,11 +570,21 @@ class DeployTask extends AbstractServerTask {
     }
 
     private String getProjectPath(File parentProjectDir, File dep) {
-        String dependencyPathPortion = dep.getAbsolutePath().replace(parentProjectDir.getAbsolutePath()+"/","")
+        String dependencyPathPortion = dep.getAbsolutePath().replace(parentProjectDir.getAbsolutePath(),"")
         String projectPath = dep.getAbsolutePath().replace(dependencyPathPortion,"")
-        Pattern pattern = Pattern.compile("/build/.*")
+        Pattern pattern
+        if (OSUtil.isWindows()) {
+            pattern = Pattern.compile("\\\\build\\\\.*")
+        } else {
+            pattern = Pattern.compile("/build/.*")
+        }
         Matcher matcher = pattern.matcher(dependencyPathPortion)
         projectPath = matcher.replaceAll("")
+        if (OSUtil.isWindows()) {
+            projectPath = projectPath.replace("\\", "")
+        } else {
+            projectPath = projectPath.replace("/", "")
```

Comment:
Why are the forward/backward slashes in the projectPath getting replaced with nothing? Is this just getting rid of a leading slash? If so, a substring might be better. If not, I don't understand how we can just get rid of all slashes in the projectPath.

Reply:
Yeah, that was just for the leading slash. I can try with a substring instead.